### PR TITLE
Maxime branch -  gazebo_ros2_bolt

### PR DESCRIPTION
**Pull request by Maxime-Fansi-laas**

### CODE REVIEW

**Comment by Maxime-Fansi-laas** [comment]
gazebo_ros2_bolt est inclu dans ros2_control_bolt

File: gazebo_ros2_bolt/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (olivier-stasse) [inline gazebo_ros2_bolt/CHANGELOG.rst]: Dear Maxime please remove this file.
It is supposed to be a summary of what are the  changes from a release to another.
If you want to keep it please follow this standard.

File: gazebo_ros2_bolt/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  controller_manager
+  gazebo_dev
+  gazebo_ros
+  hardware_interface
+  pluginlib
+  rclcpp
+  yaml_cpp_vendor
+)
+
+add_library(gazebo_hardware_plugins SHARED
```
Comment (olivier-stasse) [inline gazebo_ros2_bolt/CMakeLists.txt]: This library has the same name than the library for the gazebo_ros2_control.
Please change it to gazebo_bolt_hardware_plugins

File: gazebo_ros2_bolt/include/install/local_setup.sh [hunk 3]
```diff
@@ -0,0 +1,137 @@
+# generated from colcon_core/shell/template/prefix.sh.em
```
Comment (olivier-stasse) [inline gazebo_ros2_bolt/include/install/local_setup.sh]: Dear Maxime please remove all the install directories and their contents.

**Review by olivier-stasse — CHANGES_REQUESTED** [review]
Dear Maxime please:
- Remove all the log install build directories
- Avoid to push merge commit where probably a rebase would have been sufficient.
- Please use meaningful title  and description for your PR. "Modification from Maxime" does not tell me what your modification is doing.

**Comment by Maxime-Fansi-laas** [comment]
Deletion of install, build and log folders.
Deletion of Changelog.rst file.
In CMakeLists.txt, "gazebo_hardware_plugins"    becomes   "gazebo_bolt_hardware_plugins"

**Review by olivier-stasse — APPROVED** [review]

**Pull request merged**